mathics/builtin/atomic/strings.py

```diff
@@ -835,7 +835,16 @@ class ToString(Builtin):
     >> "U" <> ToString[2]
      = U2
     >> ToString[Integrate[f[x],x], TeXForm]
-     = \\int f\\left[x\\right] \\, dx
+     = \int f\left[x\right] \, dx
+
+    By means of the option `CharacterEncoding` we can control
+    the way in which characters are encoding:
+    >> ToString["\[Integral]f(x)\[DifferentialD]x", CharacterEncoding->"UTF-8"]
+     = ∫f(x)𝑑x
+    When the option is set to "ASCII", Unicode characters are mapped
+    to their named form.
+    >> ToString["\[Integral]f(x)\[DifferentialD]x", CharacterEncoding->"ASCII"]
+     = \[Integral]f(x)𝑑x
```

Comment:
Notice that `𝑑` is not an ASCII character, and should be (in this line) `\[DifferentialD]`. This seems to be an issue from mathics_scanner.

Reply:
Do a git blame and look at the commit log. (This is a reason why I spend time writing meaningful commit messages). 

I recall there was some back and forth on this - it was changed for a _reason_. I don't recall why and don't have the time to look it up.